controllers/common/const.go

```diff
@@ -104,7 +111,7 @@ func LoadBalancerNsName(cr *meridiov1alpha1.Trench) string {
 }
 
 func NSENsName(attr *meridiov1alpha1.Attractor) string {
```

Comment:
LoadBalancerNsName should be fixed in the same way, the name generated is "lb-fe-trench-a.trench-a.red" instead of  "lb-fe.trench-a.red"